packages/mock-block-dock/src/mock-block-dock.tsx

```diff
@@ -59,16 +66,16 @@ type MockBlockDockProps = {
  * It provides the functionality specified in the Block Protocol, and mock data which can be customized via props.
  * See README.md for usage instructions.
  * @param blockDefinition the source for the block and any additional metadata required
- * @param [blockEntity] the starting properties for the block entity
- * @param [blockInfo] metadata about the block
- * @param [blockSchema] the schema for the block entity
- * @param [debug = false] display debugging information
- * @param [hideDebugToggle = false] hide the ability to toggle the debug UI
- * @param [initialEntities] the entities to include in the data store (NOT the block entity, which is always provided)
- * @param [initialEntityTypes] the entity types to include in the data store (NOT the block's type, which is always provided)
- * @param [initialLinks] the links to include in the data store
- * @param [initialLinkedAggregations] The linkedAggregation DEFINITIONS to include in the data store (results will be resolved automatically)
- * @param [readonly = false] whether the block should display in readonly mode or not
+ * @param blockDefinition.blockEntity the starting properties for the block entity
+ * @param blockDefinition.blockInfo metadata about the block
+ * @param blockDefinition.blockSchema the schema for the block entity
+ * @param blockDefinition.debug `= false` display debugging information
+ * @param blockDefinition.hideDebugToggle `= false` hide the ability to toggle the debug UI
+ * @param blockDefinition.initialEntities the entities to include in the data store (NOT the block entity, which is always provided)
+ * @param blockDefinition.initialEntityTypes the entity types to include in the data store (NOT the block's type, which is always provided)
+ * @param blockDefinition.initialLinks the links to include in the data store
+ * @param blockDefinition.initialLinkedAggregations The linkedAggregation DEFINITIONS to include in the data store (results will be resolved automatically)
+ * @param blockDefinition.readonly `= false` whether the block should display in readonly mode or not
```

Comment:
Sidecar fix in jsdoc

Reply:
`blockDefinition` is one value on the props object - it is not the only one. This implies that there's an object called `blockDefinition` which all these things exist on, which is not the case.

This has also removed the [optional](https://jsdoc.app/tags-param.html#optional-parameters-and-default-values) indicators.

What problem were you trying to fix? I don't think this does it

Reply:
Ah yeah you are right I've made a typo. It should be fixed in https://github.com/blockprotocol/blockprotocol/pull/611/commits/e476456e59252526724dfbbb492f75743d0e1378.

I’ve been trying to resolve this notice:

<img width="879" alt="Screenshot 2022-09-15 at 18 57 47" src="https://user-images.githubusercontent.com/608862/190476704-8b61fde0-f12c-4cf6-800f-d162a8802493.png">

Reply:
[JSDoc ref](https://jsdoc.app/tags-param.html#optional-parameters-and-default-values)

Reply:
different editor implementations of JSDoc, doh 😅 
<img width="574" alt="Screenshot 2022-09-16 at 10 47 58" src="https://user-images.githubusercontent.com/37743469/190611425-86c397ce-5a97-40d5-9bd4-924ed04ee60f.png">

Reply:
Hmmm how about with this change? https://github.com/blockprotocol/blockprotocol/pull/611/commits/82c038de24245f15ee809c57bedcb275d061cc5f

Conceptually, the function has only one param, which is `props`